src/invidious/helpers/crystal_class_overrides.cr

```diff
@@ -5,7 +5,8 @@
 class TCPSocket
   def initialize(host, port, dns_timeout = nil, connect_timeout = nil, blocking = false, family = Socket::Family::UNSPEC)
     Addrinfo.tcp(host, port, timeout: dns_timeout, family: family) do |addrinfo|
-      super(addrinfo.family, addrinfo.type, addrinfo.protocol, blocking)
+      super(addrinfo.family, addrinfo.type, addrinfo.protocol)
+      Socket.set_blocking(self.fd, blocking)
```

Comment:
Please update the pverride to match the stdlib code:
https://github.com/crystal-lang/crystal/blob/abbfbf755782f35fef7e1d0a989e31225d13c423/src/socket/tcp_socket.cr#L30-L38